src/new-client/src/app/site/[site]/site-page.tsx

```diff
@@ -23,28 +23,33 @@ export const SitePage = ({ site, permissions }: SitePageProps) => {
           <p className="nhsuk-card__description">{site.address}</p>
         </div>
       </div>
-      {permissionsRelevantToCards.length > 0 && (
-        <ul className="nhsuk-grid-row nhsuk-card-group">
-          {permissionsRelevantToCards.includes('users:view') && (
-            <li className="nhsuk-grid-column-one-half nhsuk-card-group__item">
-              <Card
-                href={`${site.id}/users`}
-                title="User Management"
-                description="Assign roles to users to give them access to features at this site"
-              />
-            </li>
-          )}
-          {permissionsRelevantToCards.includes('site:manage') && (
-            <li className="nhsuk-grid-column-one-half nhsuk-card-group__item">
-              <Card
-                href={`${site.id}/attributes`}
-                title="Site Management"
-                description="Assign accessibility attributes to this site"
-              />
-            </li>
-          )}
-        </ul>
-      )}
+      <ul className="nhsuk-grid-row nhsuk-card-group">
+        <li className="nhsuk-grid-column-one-half nhsuk-card-group__item">
+          <Card
```

Comment:
Do we need to add a permissions to this card? I believe there are permissions for this or is this currently under review and therefore we need to wait?

Reply:
Yeah, the [ticket](https://nhsd-jira.digital.nhs.uk/browse/APPT-242) explicitly mentions that permissions should be done in a separate, linked [ticket ](https://nhsd-jira.digital.nhs.uk/browse/APPT-62). I'm not sure why we didn't prep these tickets together, but for whatever reason we haven't